app/decorators/application_decorator.rb

```diff
@@ -1,2 +1,39 @@
 class ApplicationDecorator < Draper::Decorator
```

Comment:
Should this live on ApplicationDecorator or a reusable module? (Doesn't block this PR)

Reply:
It's shared on both courses and providers now

Reply:
Yeah, but ultimately not all decorators will need this.

Reply:
That's true 🤷‍♂